### ISSUE TITLE
accessibility-colorblind options

### ISSUE THREAD

**Issue by nergiszaim** [rstudio/bslib#358]
hi!

Is there a theme for accessibility? In particular, I am looking for color blind options in `bs_theme`

Thank you!
Nergis

**Comment by cpsievert** [comment]
I'm not aware of one, but I do know that Bootstrap itself is doing more to be a11y compliant (i.e., Bootstrap 5 should be a bit better than Bootstrap 4 when it comes to things like color contrast, etc)
